### PR TITLE
Wish Image and Email Service

### PR DESCRIPTION
**Pull request by ritiksr25**
- Personalised image for wishes
- Upgraded email service

### CODE REVIEW

**Review by rhnmht30 — APPROVED** [review]
LGTM! 
Is this rigorously tested with long names? can you share some screenshots?

File: services/emailService.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	let mailOptions = {
+		from: `DSCKIET <${EMAIL_SENDER}`,
+		to: `${data.name} <${data.email}>`,
+		cc: "dsckiet@gmail.com",
+		subject: WISH_MAIL_SUBJECT,
+		html: generateMailHtml(toTitleCase(String(name).trim().split(" ")[0])),
+		attachments: [
+			{
+				filename: `wish_${data._id}.png`,
+				path: imagePath,
```
Comment (rhnmht30) [inline services/emailService.js:42]: What would be the value of `imagePath` here? How is the image being uploaded and accessible?

**Comment by ritiksr25** [comment]
> LGTM!
> Is this rigorously tested with long names? can you share some screenshots?

no issues with long names, we are using first name only

File: services/emailService.js (hunk 1 above)
Comment (ritiksr25) [reply services/emailService.js:42]: - value of imagePath is path to assets/images/wish_<userid>.png file.
- for every user, an image is being generated with his name and image using node canvas and being saved as wish_<userid>.png in assets/images folder

Comment (rhnmht30) [reply services/emailService.js:42]: so is there a way to not save files and use the generated image as it is in the attachment?

Comment (ritiksr25) [reply services/emailService.js:42]: will be sending only buffer

**Pull request merged**